modules/pushapk_scriptworker/files/requirements.txt

```diff
@@ -71,4 +71,4 @@ virtualenv==16.1.0
 voluptuous==0.11.5
 wcwidth==0.1.7
 yarl==1.3.0
-pushapkscript==0.9.0  # puppet: nodownload
+pushapkscript==0.10.0  # puppet: nodownload
```

Comment:
Unsure why just `pushapkscript` is `puppet: nodownload`'d :thinking:

Reply:
Because `mozapkpublished` is on pypi (https://pypi.org/project/mozapkpublisher/#history) and pushapkscript isn't, so we don't have puppet download from pypi (storing local copies) of the latter.

Reply:
Hmmm, we could probably `puppet: nodownload`'d `mozapkpublisher` too. I don't remember what's the value added by this check. Would you have some pointer @escapewindow ?

Reply:
FWIW, @Callek, both packages are on Pypi:
* [mozapkpublisher](https://pypi.org/project/mozapkpublisher/)
* [pushapkscript](https://pypi.org/project/pushapkscript/)

Reply:
`# nodownload` is to avoid trying to download packages that aren't on pypi. If both of them have their latest releases on pypi and will continue to do so, we can remove the `# nodownload`.